### ISSUE TITLE
Error: getaddrinfo ENOTFOUND when trying to  connect to a server

### ISSUE THREAD

**Issue by eugenlisov** [icetee/remote-ftp#745]
Hi,

I'm having a big issue with connecting to a FTP server with remote-ftp. 

It looks like this:
'Remote FTP: Connection failed
Error: getaddrinfo ENOTFOUND ftp://ftp.lumeaadidasilor.ro ftp://ftp.lumeaadidasilor.ro:21". I couln't find anything related to this anywhere.

It's only happening to one hosting provider ( I tried two different shared accounts from them, same issue ).
I tried discussing it with them, but not much help. They said "Eliminate the  TLS from the configuration and try again". I don't know how I could do that or if that's the issue.

It's definitely some particular server configuration they have and  Atom can't get around. I didn't have that issue anywhere else (I've worked with Atom on tens of servers). Also, only Atom has a problem on this provider. Connection works with  other ftp clients ( FileZilla, WinSCP, Dreamweaver, web ftp etc). 

If you have the time, You canreplicate the issue with this temp FTP I created.
host: ftp.lumeaadidasilor.ro
u: testftp@lumeaadidasilor.ro
p:TestingPassword123!

I hope to get this fixed, otherwise I'm forced to work with something else than Atom ( which would really suck) or ditch the host altogether (would suck less, but still quite inconvenient).

The FTP config file looks like this:

{
    "protocol": "ftp",
    "host": "ftp://ftp.lumeaadidasilor.ro",
    "port": 21,
    "user": "testftp@lumeaadidasilor.ro",
    "pass": "TestingPassword123!",
    "promptForPass": false,
    "remote": "/",
    "local": "",
    "secure": false,
    "secureOptions": null,
    "connTimeout": 10000,
    "pasvTimeout": 10000,
    "keepalive": 10000,
    "watch": [],
    "watchTimeout": 500
}

Thank you very much!

**Comment by icetee** [comment]
Not correct host param. Try simple "ftp.lumeaadidasilor.ro".

**Comment by thedancollins** [comment]
Was getting this error - the required change was to remove a trailing slash (/) from the host.  sftp.whatever.net instead of sftp.whatever.net/

**Comment by eugenlisov** [comment]
Hey! Thanks for the replies.
I tried all and still nothing. I simply switched to a different host and that was it.

**Comment by bearpaw** [comment]
I'm getting the same issue with the same error
```
Error: getaddrinfo ENOTFOUND gateway2 gateway2:21

```

The `~/.ssh/config` file is like this
```
Host gateway
    HostName gateway.xx.com
    User myname
    Port 22
    IdentityFile /home/myname/.ssh/id_rsa
```

And the `.ftpconfig` file is like this
```
{
    "protocol": "sftp",
    "host": "gateway"
}
```

Can anybody help please?

**Comment by icetee** [comment]
@bearpaw Maybe add remote property? `"remote": "/"`
And `ssh gateway` works in command line?

**Comment by bearpaw** [comment]
@icetee Thanks. I added but it did not work. 

Yes, `ssh gateway` works in terminal. And my OS is ubuntu 16.04.

**Comment by icetee** [comment]
The plugin want to connect to 21 port, but I can see in ssh config file. Try add `.ftpconfig` temporarily. https://github.com/icetee/remote-ftp/commit/683bc17b430f7a32ee7ed6cd0d15b0adb5ee843e

**Comment by bearpaw** [comment]
Thanks. The problem still exists on my laptop. I will try it on other desktops and see whether this problem repeats.

**Comment by baker13** [comment]
Hi guys I got this issue as well. But I discovered that the reason behind the issue "for me" was my network line that I was on. I was connected to my work network through an ether-net cable. And I'm not sure exactly what the issue is but I assume it's because they have network restrictions. So that staff don't use the network for nonsense like games.

As soon as I connected to a different network my error went away. 

I tried ssh for this issue and it did not solve anything. Changing to an unrestricted network is what fixed the issue for me. :) 

I hope this helps someone :)
